script/_dhcpv6_pd_ref

```diff
@@ -30,39 +30,110 @@
 #       This script manipulates DHCPv6-PD-REF configuration.
 #
 
-customise_dhcpcd_conf()
+DHCPCD_ENTER_HOOK="/etc/dhcpcd.enter-hook"
+DHCPCD_EXIT_HOOK="/etc/dhcpcd.exit-hook"
+
+PD_DAEMON_DIR="/opt/pd-daemon"
+PD_DAEMON_PATH="${PD_DAEMON_DIR}/dhcp6_pd_daemon.py"
+PD_DAEMON_SERVICE_NAME="dhcp6_pd_daemon.service"
+PD_DAEMON_SERVICE_PATH="/etc/systemd/system/${PD_DAEMON_SERVICE_NAME}"
+
+DHCP_CONFIG_FILE="/etc/dhcpcd.conf.orig"
+DHCP_CONFIG_ORIG_FILE="/etc/dhcpcd.conf.orig"
+DHCP_CONFIG_WITH_PD_FILE="/etc/dhcpcd.conf.with-pd"
+DHCP_CONFIG_NO_PD_FILE="/etc/dhcpcd.conf.no-pd"
+
+# Create dhcpcd configuration file with ipv6 prefix request.
+create_dhcpcd_conf_with_pd()
 {
-    # This has to be run after script/_border_routing, and this will
-    # invalidate all changes to dhcpcd.conf made by script/_border_routing.
-    sudo tee /etc/dhcpcd.conf >/dev/null <<EOF
+    sudo tee ${DHCP_CONFIG_WITH_PD_FILE} >/dev/null <<EOF
 noipv6rs # disable router solicitation
 interface eth0
   iaid 1
   ia_pd 2/::/64 -
+release
+noipv6rs
```

Comment:
There're 2 `noipv6rs` lines. Are they necessary?

Reply:
Yes. Without the second noipv6rs, accept_ra is set zero. However, there is no documentation talking about it. It's just experimental finding

Reply:
That's very surprising. Is there a configuration that can achieve the same behavior without multiple `noipv6rs`?

Alternatively, I'm fine if you keep both `noipv6rs` and document clearly why they are needed.